sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryErrorsBase.scala

```diff
@@ -92,10 +84,6 @@ private[sql] trait QueryErrorsBase {
     quoteByDefault(text.toUpperCase(Locale.ROOT))
   }
 
-  def toSQLConf(conf: String): String = {
```

Comment:
it's a bit weird that we put this in `DataTypeErrorsBase`, shall we name it `QuotingUtils`? And we can move methods to it, like `toSQLType`

Reply:
done